pkg/server/status/runtime.go

```diff
@@ -280,4 +450,118 @@ func (rsr *RuntimeStatSampler) SampleEnvironment(ctx context.Context) {
 	rsr.FDSoftLimit.Update(int64(fds.SoftLimit))
 	rsr.Rss.Update(int64(mem.Resident))
 	rsr.Uptime.Update((now - rsr.startTimeNanos) / 1e9)
+
+	if err := rsr.recordDiskStats(ctx); err != nil {
```

Comment:
Nit: these methods aren't actually recording disk stats (which typically means writing to disks), they're sampling these statistics.  The word "sample" is used for other methods on this class (e.g. "SampleEnvironment"), so I would suggest using "sample" as the prefix instead to avoid confusion.

Reply:
Done